timeseries/src/autogluon/timeseries/predictor.py

```diff
@@ -161,7 +145,6 @@ def __init__(
         log_file_path: Union[str, Path] = "auto",
         quantile_levels: Optional[List[float]] = None,
         cache_predictions: bool = True,
-        learner_type: Optional[Type[AbstractLearner]] = None,
         learner_kwargs: Optional[dict] = None,
```

Comment:
Should we also deprecate `learner_kwargs` while we are at it? It seems that pretty much all the arguments are simply forwarded from the predictor to the learner, I don't see why anyone would want to override them.